### DISCUSSION TITLE
Json Table

### DISCUSSION THREAD

**Discussion by Supermario-21** [Zefau/ioBroker.jarvis#2365] [Q&A]
Hallo, ich hab leider nicht viel zu Json Table gefunden.

Ich suche eine Möglichkeit in der Tabelle Texte farbig zu formatieren oder auch Icons anzuzeigen? Kann dies vielleicht bereits im ioBroker-Json-Datenpunkt mitgeben werden oder im Jarvis-Gerät? Wie müsste der Code aussehen?!

![image](https://github.com/Zefau/ioBroker.jarvis/assets/104907692/6b42c6d1-443d-4108-afef-11c3bd00c871)

Ich hab es erst mit HTML-Tabellen versucht, dort ist die Aktualisierung bei Wertänderungen sehr sporadisch und nicht wirklich sicher.

**Comment by mcuiobroker** [comment]
In jedem Json-Objekt kann ein HTML Code # genutzt werden. 
```Z.b "<span style='color: red;'> test </span>"```

Die HTMLTable und auch die JSONTable werden verändert sobald der DP-Wert sich ändert, in dem sie drin stehen.

Wie erstellst du die JSONTable? javascript, blokcly -> reinstellen

**Reply by Supermario-21** [reply]
Das hört sich gut an! Leider hab ich keine Ahnung, wo und wie ich das in Json mitgeben muss, damit Jarvis es richtig interpretiert. Ich erstell den Inhalt über über Blockly.

hier ein Auszug:
  {
    "SW": "EG",
    "Name": "Wintergarten ",
    "S": "VH",
    "Offen": "14:10"
  },

Tabelle in Jarvis
![image](https://github.com/Zefau/ioBroker.jarvis/assets/104907692/e4f03754-53fa-4ba0-9aeb-9af3c23b0650)

Wie würde ich die "14:10" rot formatieren?
und den Punkt einfügen
<font color=red>◉</font> 

Danke im Voraus!

**Reply by mcuiobroker** [reply]
```
"Offen": "<span style='color:red;'>14:10 ◉</span>"
oder
"Offen": "<div style='color:red;'>14:10 ◉</div>"
```

**Reply by Supermario-21** [reply]
Hab ich gemacht: 
![image](https://github.com/Zefau/ioBroker.jarvis/assets/104907692/ad2e6de8-65b3-47a4-aca9-d920b5b7d474)


Ergebnis:
![image](https://github.com/Zefau/ioBroker.jarvis/assets/104907692/2b810ac7-e545-4479-a0ee-b0b1d7ffd357)
![image](https://github.com/Zefau/ioBroker.jarvis/assets/104907692/bc667815-9863-4719-9024-2da179ff0439)


An was kann das liegen?
Muss ich "Pro" sein?

**Reply by mcuiobroker** [reply]
Ich glaube schon, da HTML nur in PRO genutzt werden kann. siehe oben fixierte Diskussionen.
![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/5dd9fc2b-3751-4e3d-bc45-684f8863e4dd)
